terragrunt/env/terragrunt.hcl

```diff
@@ -0,0 +1,52 @@
+locals {
+  account_id       = "545719101351"
+  domain           = ""
+  env              = "dev"
+  product_name     = "next-auth-sandbox-wm"
+  cost_center_code = "${local.product_name}-${local.env}"
+}
+
+# DO NOT CHANGE ANYTHING BELOW HERE UNLESS YOU KNOW WHAT YOU ARE DOING
+
+inputs = {
+  account_id                = local.account_id
+  domain                    = local.domain
+  env                       = local.env
+  product_name              = local.product_name
+  region                    = "ca-central-1"
+  billing_code              = local.cost_center_code
+}
+
+generate "provider" {
+  path      = "provider.tf"
+  if_exists = "overwrite"
+  contents  = file("./common/provider.tf")
+
+}
+
+generate "common_variables" {
+  path      = "common_variables.tf"
+  if_exists = "overwrite"
+  contents  = file("./common/common_variables.tf")
+}
```

Comment:
If you're curious, these are the 2 blocks that will take care of regenerating those files that can be deleted from the PR.